content/04.results.md

```diff
@@ -41,3 +41,19 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Discriminative Model Calibration
+
+![
+Calibration plots for the discriminative model.
+A perfectly calibrated model would follow the dashed diagonal line.
+The blue line represents the predictions after calibration and the green line shows predictions after calibration. 
+](https://raw.githubusercontent.com/danich1/snorkeling/86037d185a299a1f6dd4dd68605073849c72af6f/figures/model_calibration_experiment/model_calibration.png){#fig:discriminative_model_calibration}
+
+Even high performing deep learning models are subject to poor calibration. Typically, these models are overconfident in their predictions [@arxiv:1706.04599; @arxiv:1807.00263].
+We accounted for this problem by calibrating our best performing discriminative models.
+Results of the calibration process are reported in Figure {@fig:discriminative_model_calibration}.
+Before calibration (green line) we found that the all edge types were somewhat aligned with the ideal calibration lines when predicting low probability scores (close to 0.25).
+After applying the calibration algorithm (blue line), we found that there wasn't a drastic change in predictions.
+An exception to this would be Disease associates Gene (DaG) where high predicting probabilities were brought closer to the calibration line.
+Overall, Compound binds Gene (CbG) and Gene interacts Gene (GiG) model calibration failed to show any significant improvement, while DaG and Compound Treats Disease (CtD) models show some improvement in the higher scoring sections (0.75-1).
```

Comment:
```suggestion
```
I don't think this sentence really helps too much. I'm not sure what the best way to close out this section would be though. Probably this will end up back in the supplement. How did you run calibration for some of these models? I thought they weren't going to be done until next Tuesday.

Reply:
The Tuesday delay was trying to finish running the discriminator model on the 16 LF sample for GiG. The discriminator model for the 28 LF sample point already finished. Since it finished early, I was able to complete downstream analyses while the 16 LF point was still processing.